acceptance/chaos_test.go

```diff
@@ -19,35 +19,43 @@
 package acceptance
 
 import (
+	"bytes"
+	"database/sql"
+	"flag"
 	"fmt"
-	"net/rpc"
+	"math/rand"
 	"strings"
 	"sync"
 	"sync/atomic"
 	"testing"
 	"time"
 
-	"github.com/cockroachdb/cockroach/client"
-	"github.com/cockroachdb/cockroach/roachpb"
 	"github.com/cockroachdb/cockroach/testutils"
 	"github.com/cockroachdb/cockroach/util/log"
 	"github.com/cockroachdb/cockroach/util/randutil"
 	"github.com/cockroachdb/cockroach/util/stop"
 )
 
-// TestChaos starts up a cluster and, for each node, a worker writing to
-// independent keys, while nodes are being killed and restarted continuously.
-// The test measures not write performance, but cluster recovery.
+var maxTransfer = flag.Int("max-transfer", 999, "Maximum amount to transfer in one transaction.")
+var numAccounts = flag.Int("num-accounts", 999, "Number of accounts.")
+
+// TestChaos starts up a cluster with an "accounts" table.
+// It starts transferring money between accounts, while nodes are
+// being killed and restarted continuously.
+// The test doesn't measure write performance, but cluster recovery.
+// TODO(vivek): Expand this test to check that write performance
+// is unaffected by chaos.
 func TestChaos(t *testing.T) {
 	c := StartCluster(t)
 	defer c.AssertAndStop(t)
 
 	num := c.NumNodes()
+	if num <= 0 {
+		log.Fatal("%d nodes in cluster", num)
```

Comment:
TIL. We should do that - whatever complexities exist wrt `go vet`/`go tool vet` are already handled in the makefile